inc/class-customizer-guided-tour.php

```diff
@@ -0,0 +1,234 @@
+		 * Customizer.
+		 *
+		 * @since 0.8.8
+		 */
+		public function customizer() {
+			global $pagenow;
+
+			// Always load the assets when in the customizer.
+			if ( 'customize.php' === $pagenow ) {
+				add_action( 'customize_controls_enqueue_scripts',      array( $this, 'customize_scripts' ) );
+				add_action( 'customize_controls_print_footer_scripts', array( $this, 'print_templates' ) );
+				if ( isset( $_GET['store-wpcom-nux'] ) ) {
+					$this->isStorefrontActivated();
+				}
+			}
+		}
+
+		public function isStorefrontActivated() {
+			$current_theme = wp_get_theme();
+
+			if ( 'Storefront' == $current_theme->get( 'Name' ) ) {
+				$redirect_url = add_query_arg( array(
+					'sf_guided_tour' => '1',
+					'sf_tasks' => 'homepage',
+					'theme' => $_GET['theme'],
+				), admin_url( '/customize.php' ) );
+
+				wp_redirect( $redirect_url );
+				exit;
+			}
+		}
+
+		/**
+		 * Customizer enqueues.
+		 *
+		 * @since 0.8.8
+		 */
+		public function customize_scripts() {
+			global $storefront_version;
+
+			wp_enqueue_style( 'wc-store-nux-tour', WC_API_Dev::$plugin_url . 'assets/css/admin/customizer.css', array(), $storefront_version, 'all' );
+
+			wp_enqueue_script( 'wc-store-nux-tour', WC_API_Dev::$plugin_url . 'assets/js/admin/customizer.js', array( 'jquery', 'wp-backbone' ), $storefront_version, true );
+
+			wp_localize_script( 'wc-store-nux-tour', '_wpStoreNuxTourSteps', $this->guided_tour_steps() );
+			wp_localize_script( 'wc-store-nux-tour', '_wpStoreNuxSettings', $this->guided_tour_settings() );
+		}
+
+		/**
+		 * Template for steps.
+		 *
+		 * @since 0.8.8
+		 */
+		public function print_templates() {
+			?>
+			<script type="text/html" id="tmpl-wc-store-tour-step">
+				<div class="sf-guided-tour-step">
+					<a class="sf-guided-tour-close" href="#">
+						<span class="dashicons dashicons-no-alt"></span>
+					</a>
+					<# if ( data.title ) { #>
+						<h2>{{ data.title }}</h2>
+					<# } #>
+					{{{ data.message }}}
+					<# if ( data.altStep ) { #>
+						<a class="sf-nux-button sf-nux-alt-button {{ data.altStep.className }}" href="#">
+							{{ data.altStep.buttonText }}
+						</a>
+					<# } #>
+					<# if ( data.buttonText ) { #>
+						<a class="sf-nux-button sf-nux-primary-button {{ data.className }}" href="#">
+							{{ data.buttonText }}
+						</a>
+					<# } #>
+
+					<# if ( data.showSkip ) { #>
+						<a class="sf-guided-tour-skip" href="#">
+							<?php esc_attr_e( 'Skip this step', 'wc-api-dev' ); ?>
+						</a>
+					<# } #>
+				</div>
+			</script>
+			<?php
+		}
+
+		/**
+		 * Guided tour settings.
+		 *
+		 * @since 0.8.7
+		 */
+		public function guided_tour_settings() {
+			$show_tour = isset( $_GET['store-wpcom-nux'] );
+			$theme_supports_woo = current_theme_supports( 'woocommerce' );
+			$supported_themes = wc_get_core_supported_themes();
+			$current_theme = get_template();
+
+			return array(
+				'autoStartTour' => ( bool ) $show_tour,
+				'showTourAlert' => ( bool ) ! $show_tour,
```

Comment:
It looks like we are missing some logic here still -- the alert should only show if `$current_theme` is not in `$supported_themes`.

Additionally, I think you can check `$_GET['theme']` to see if it equals `Storefront` and also hide the alert in this case. That query string means they clicked your notice and are previewing `Storefront` in the customizer.

Reply:
Looks like my fix for this in b153878 was lost in the conflict merge above - I will add it back in.